### PR TITLE
Phillip Stewart

### PR DESCRIPTION
**Pull request by philmarcu**
Finish Iterations 1-4

### CODE REVIEW

**Comment by philmarcu** [comment]
Question - Why does sort() work that way in the sorted by room method? I did sort_by .reverse after the fact and it worked as well, but I'm confused on why sort works without reversing it. Isn't sort() usually supposed to go from least <> greatest unless you make a <=> operator and specify otherwise?

File: spec/house_spec.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    house.add_room(room_3)
+    house.add_room(room_4)
+
+    expected = {
+      :bedroom => [room_1, room_2],
+      :living_room => [room_3],
+      :basement => [room_4]
+    }
+
+    expect(house.rooms_by_category).to eq(expected)
```
Comment (cjsim89) [inline spec/house_spec.rb:86]: I see this method is tested, but it's not what's asked in the spec which is `rooms_from_category(...)`. You do have a method for that so it counts (and it works 👍 ), but make sure you have tests for that method as well.

File: spec/house_spec.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
+require './lib/room'
+require './lib/house'
+
+RSpec.describe House do
+  let(:house) {House.new("$400000", "123 sugar lane")}
+  let(:room_1) {Room.new(:bedroom, 10, '13')}
+  let(:room_2) {Room.new(:bedroom, 11, '15')}
+  let(:room_3) {Room.new(:living_room, 25, '15')}
+  let(:room_4) {Room.new(:basement, 30, '41')}
```
Comment (cjsim89) [inline spec/house_spec.rb:9]: What do you find different between using `before :each do ` and `let`..?

**Comment by philmarcu** [comment]
I mostly used let so I didn't have to type the @ sign out a bunch. Isn't
the rooms_from_category being tested in my spec on line 30?

**Pull request closed without merging**